[Zurb] Fixes positioning issue of labels in IE11

floriank opened the pull request:
Meets https://community.openproject.org/work_packages/19046

Commit:
fixes positioning issue of labels in IE11

Meets https://community.openproject.org/work_packages/19046

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://floriank-fix-19046-custom-field-checkboxes.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://floriank-fix-19046-custom-field-checkboxes.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

ulferts commented:
@floriank I know I started down the wrong path with the padding business. Thinking about your fix however made me realise, that this is the wrong way and increasing the padding makes it look ugly IMO.

I played around a bit and I think that the correcter<sup>TM</sup> fix is to apply `display: block` to the container. That will break when a second subelement is applied but that might be a problem for futur @floriank. 

The padding fix will break as soon as an input with a different width is added.

The correct fix is probably checking how flex growth/shrink works in IE11 or take a look at box-sizing.

Commit:
use display: block approach to fix label issue

This will make use of setting the container div to a block display and resolve
the padding issue in the trailing-label context.

However, this will break flexbox-related content within such containers, e.g.
using .form--field-inline-action.

ulferts commented on app/assets/stylesheets/content/_forms.sass:
```diff
@@ -232,12 +232,17 @@ fieldset.form--fieldset
       // override max-width set by including grid-content
       max-width: 100%
-      padding-left: 0.5rem
+      padding: 0.5rem
```
@floriank I don't think that you need to apply the padding to anything but `left`

floriank replied:
This seems to me like i was not paying attention. I honestly cannot remember changing this.

ulferts replied:
Well, can we change it back because I find it weird that we would need a top/border padding in this case.

Commit:
fix padding

[ci skip]
